portal/views/role.py

```diff
@@ -6,7 +6,7 @@
 from ..models.role import Role
 from ..models.user import current_user, get_user_or_abort
 
-role_api = Blueprint('role_api', __name__,)
```

Comment:
Cleaner to remove the trailing comma :)

Reply:
This should be resolved although github is not collapsing the comment